spec/features/user_add_review_spec.rb

```diff
@@ -1,36 +1,49 @@
 require 'rails_helper'
+include Warden::Test::Helpers
 
-feature "visitors can add reviews for the location" do
-  scenario "adds a review for the location successfully" do
-    dumpling_king = Location.create(name_of_location: 'Dumpling King', description: "Great place for a cheap date, but not a first date")
+feature 'authenticated users can add reviews for the location' do
+  let!(:user) { FactoryGirl.create(:user) }
+  let!(:location) { FactoryGirl.create(:location) }
 
-    visit location_path(dumpling_king)
+  scenario 'an authenticated user can navigate to the review form' do
+    login_as(user)
+    # dumpling_king = Location.create(name_of_location: 'Dumpling King', description: 'Great place for a cheap date, but not a first date')
```

Comment:
take out commented out code. it's not doing anything besides potentially confusing you and your team